node/metrics.go

```diff
@@ -177,3 +177,17 @@ func makeJunoMetrics(version string) {
 		ConstLabels: prometheus.Labels{"version": version},
 	}))
 }
+
+func makeBlockchainMetrics() blockchain.EventListener {
+	reads := prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "bc",
```

Comment:
maybe `chain` would be more self-describing than `bc`

Reply:
changed it to `blockchain`